core/src/main/java/greencity/controller/UserController.java

```diff
@@ -1180,4 +1175,26 @@ public ResponseEntity<Object> markUserAsDeactivated(
         userService.markUserAsDeactivated(uuid);
         return ResponseEntity.status(HttpStatus.OK).build();
     }
+
+    /**
+     * Controller edit an employee`s authorities.
+     *
+     * @return {@link UserEmployeeAuthorityDto}
+     *
+     * @author Hlazova Nataliia.
```

Comment:
Due to our convention and English language, its better to write first name first and then last name 
```suggestion
     * @author Nataliia Hlazova.
```